README.md

```diff
@@ -8,7 +8,7 @@ This repository contains a script that can be used to easily install a PHP exten
 
 The script will install all the required APT/APK packages; at the end of the script execution, the no-more needed packages will be removed so that the image will be much smaller.
 
-Supported docker images are all the Alpine/Debian versions, except for PHP 5.5 where we only support Debian 8 (jessie) (that is, `php:5.5`, `php:5.5-apache`, `php:5.5-cli`, `php:5.5-fpm`, `php:5.5-zts`).
+Supported docker images are all the Alpine/Debian versions, except for PHP 5.5 where we only support Debian 8 (jessie).
```

Comment:
Any reason to change this line?

Reply:
I think it is not needed... I would even drop support for PHP 5.5 ;-)

Reply:
No, I won't drop support for PHP 5.5. And I don't see any problem being more explicit about supported versions (it may also save me some time because I won't have to answer questions about exactly supported versions)

Reply:
I removed this change...